src/sentry/api/endpoints/sentry_apps_stats.py

```diff
@@ -21,7 +22,14 @@ def get(self, request: Request) -> Response:
             sentry_apps = sentry_apps[: int(request.query_params["per_page"])]
 
         apps = [
-            {"id": app.id, "slug": app.slug, "name": app.name, "installs": app.installations__count}
```

Comment:
The `serialize()` function can actually take a list of object. This will let you combine all of the DB requests.
```py
def serialize(
    objects: Union[Any, Sequence[Any]],
    user: Optional[Any] = None,
    serializer: Optional[Any] = None,
    **kwargs: Any,
) -> Any:
    ...
```
It would looks something like:
```py
"avatars": serialize(app.avatar.all())
```

Reply:
I noticed that this was actually making a DB call for every app, and so was the SentryApp serializer. I've refactored them into another manager with a method that will do the defaultdict mapping thing to reduce the DB calls.